jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -988,6 +988,17 @@ pipeline {
                 script {
                     if (params.RC_NUMBER.toInteger() > 0) {
                         triggerDistributionValidationWorkflow(env.revision)
+                        triggerNightlyPlayground()
+                    }
+                    postCleanup()
+                }
+            }
+        }
+        unstable {
```

Comment:
why unstable?

Reply:
Because we mark a stage unstable if build was successful but some plugins failed (continue on error)
https://build.ci.opensearch.org/view/Build/job/distribution-build-opensearch/

Code: https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L1018-L1022